makefiles/tailwindcss.mk

```diff
@@ -0,0 +1,57 @@
+# Variables
+TW_VERSION := 3.4.13
+TW_BASE_URL := https://github.com/tailwindlabs/tailwindcss/releases/download/v$(TW_VERSION)
+TW_CHECKSUM_FILE := sha256sums.txt
```

Comment:
I think we should commit this file in the repo, otherwise we can not be sure of its integrity (unless the tailwind project would sign it and we could verify its integrity that way). It should be updated when we bump the version here.